db/sql/blocks/get_common_block_by_previous_block.sql

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+
+/*
+  DESCRIPTION: ?
```

Comment:
Description may be removed. The title is descriptive enough.

Reply:
Its consistent across all files. Should we remove from all files?

Reply:
@MaciejBaj Should we do it in this PR? or we update comments in SQL files for all files in other PR?